diff --git a/lib/add-channel.js b/lib/add-channel.js
index 7dfdaab6e2b8c517e8d706b33461a2769f0783a5..ae70cdc010755d50500d21f10cd2043a215e4249 100644
--- a/lib/add-channel.js
+++ b/lib/add-channel.js
@@ -19,7 +19,7 @@ export default async function (npmrc, { npmPublish }, pkg, context) {
 
     logger.log(`Adding version ${version} to npm registry on dist-tag ${distTag}`);
     const result = execa(
-      "npm",
+      "pnpm",
       ["dist-tag", "add", `${pkg.name}@${version}`, distTag, "--userconfig", npmrc, "--registry", registry],
       {
         cwd,
diff --git a/lib/definitions/errors.js b/lib/definitions/errors.js
index 180b1bf60a013051983bff0cbfc38ed0476bceb2..6b75403e34a7cc9391869d9eb370f2cd9dcd807a 100644
--- a/lib/definitions/errors.js
+++ b/lib/definitions/errors.js
@@ -1,11 +1,4 @@
-import { dirname, resolve } from "node:path";
-import { fileURLToPath } from "node:url";
-import { readPackageSync } from "read-pkg";
-
-const __dirname = dirname(fileURLToPath(import.meta.url));
-const pkg = readPackageSync({ cwd: resolve(__dirname, "../../") });
-const [homepage] = pkg.homepage.split("#");
-const linkify = (file) => `${homepage}/blob/master/${file}`;
+const linkify = (file) => `https://github.com/semantic-release/npm/blob/master/${file}`;
 
 export function EINVALIDNPMPUBLISH({ npmPublish }) {
   return {
diff --git a/lib/prepare.js b/lib/prepare.js
index 3e76bec44cf595a1b4141728336bed904d4d518d..139b0b9ceab52e824915b21a2ac9cb7ffda0afd6 100644
--- a/lib/prepare.js
+++ b/lib/prepare.js
@@ -12,7 +12,7 @@ export default async function (
   logger.log("Write version %s to package.json in %s", version, basePath);
 
   const versionResult = execa(
-    "npm",
+    "pnpm",
     ["version", version, "--userconfig", npmrc, "--no-git-tag-version", "--allow-same-version"],
     {
       cwd: basePath,
diff --git a/lib/publish.js b/lib/publish.js
index ac3e6feea8e388d88e77d58538a265f7ab4eac05..dd5653348713eeb26e89681421b6524e27f83623 100644
--- a/lib/publish.js
+++ b/lib/publish.js
@@ -21,7 +21,7 @@ export default async function (npmrc, { npmPublish, pkgRoot }, pkg, context) {
 
     logger.log(`Publishing version ${version} to npm registry on dist-tag ${distTag}`);
     const result = execa(
-      "npm",
+      "pnpm",
       ["publish", basePath, "--userconfig", npmrc, "--tag", distTag, "--registry", registry],
       { cwd, env, preferLocal: true }
     );
diff --git a/lib/verify-auth.js b/lib/verify-auth.js
index 9b9f921dfd386a53031c76fa501371c4445672ce..53f13df0972c754a8eeaea5495c2ad282e8c7022 100644
--- a/lib/verify-auth.js
+++ b/lib/verify-auth.js
@@ -38,7 +38,7 @@ export default async function (npmrc, pkg, context) {
 
   if (registryIsDefault(registry, DEFAULT_NPM_REGISTRY)) {
     const publishDryRunResult = execa(
-      "npm",
+      "pnpm",
       ["publish", "--dry-run", "--tag=semantic-release-auth-check", "--userconfig", npmrc, "--registry", registry],
       { cwd, env, preferLocal: true, lines: true }
     );
